src/dataexplorer/dataexplorer-application/src/types/IPatient.ts

```diff
@@ -3,18 +3,44 @@ export interface IPatient {
     name: IName[];
     birthDate: string;
     gender: string;
+    map(data: any): any;
+    address: IAddress[];
+    telecom: ITelecom[];
+    photo: IPhoto[];
+    url: string;
 }
 
-export interface IName {
-    family: string;
-    given: string[];
+export interface IState {
+    setName: (name: string) => void;
+    setSchema: (id: string) => void;
+    patientID: number;
+}
```

Comment:
State is a very generic expression in React, I think this should be renamed. 

Because this is an interface for props in a component I think this interface could actually be put in the same file as the component and be named IPatientInfoProps

Reply:
Sure, Ill rename it. 

Ok. I thought we wanted all interfaces to be located in one place, but I can move it if that is wanted (Please confirm, I don't know the standard for interfaces at all)?

Reply:
Yes, for all interfaces except interfaces just used for props and not for objects. Understandable this is confusing